usmqe_tests/web/others/test_import_cluster.py

```diff
@@ -90,7 +46,33 @@ def test_initial_import_cluster(valid_credentials):
 
     # Check that cluster is present in the list
     NavMenuBars(valid_credentials.driver).open_clusters(click_only=True)
-    cluster_list = ClustersList(valid_credentials.driver)
-# TODO: Check that correct cluster is present in the list
-    pytest.check(len(cluster_list) == 1,
+    clusters_list = ClustersList(valid_credentials.driver)
+
+    pytest.check(len(clusters_list) == 1,
                  'There should be exactly one cluster in tendrl')
+    present = False
+    for cluster in clusters_list:
+        if cluster_ident.lower() in cluster.name.lower():
+            present = True
+            break
+    pytest.check(present,
+                 'The imported cluster should be present in the cluster list')
+
+
+def test_import_cluster(valid_credentials):
+    """
+    positive import cluster test
+
+    NOTE: 1. Some cluster has to be already present in the list
```

Comment:
Yes, if there is some cluster, landing page is different and so the approach how to import a cluster differs too.